src/metabase/api/search.clj

```diff
@@ -364,16 +365,16 @@
   true)
 
 (defmethod check-permissions-for-model :metric
-  [{:keys [id]}]
-  (-> (db/select-one Metric :id id) mi/can-read?))
+  [o]
+  (mi/can-read? o))
```

Comment:
We're using different names all over the place to mean "row" or "instance". I've also seen: `entity`, `object`, `obj`, `o`, and the list goes on.

Let's standardize on calling an instance of a Toucan model an _instance_

```suggestion
  [instance]
  (mi/can-read? instance))
```